app/controllers/users_controller.rb

```diff
@@ -5,9 +5,10 @@ class UsersController < ApplicationController
   GROUP_MESSAGING_ENABLED = "1"
 
   before_action :set_user, only: %i[show edit update]
+  before_action :authenticate_user!
```

Comment:
@hectorcorrea This line is redundant, and I think it will add confusion https://github.com/pulibrary/pdc_describe/blob/aa589ca8600480114ac307bf5155334433ea229e/app/controllers/application_controller.rb#L10

Can you remove it from the PR?

Reply:
@carolyncole good point. I added a comment to the code to indicate why we need both.